### PR TITLE
add maxNumberMessagesToFetch to docs

### PR DESCRIPTION
**Pull request by HugoPct**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

File: content/en/synthetics/private_locations/configuration.md [hunk 1]
```diff
@@ -99,6 +99,11 @@ Discard SSL errors when private location is using a proxy to send requests to Da
 **Default**: `10`<br>
 Maximum number of tests executed in parallel.
 
+`maxNumberMessagesToFetch`
+: **Type**: Number <br>
+**Default**: `10`<br>
+Maximum number of tests fetched from Datadog.
```
Comment (buraizu) [inline content/en/synthetics/private_locations/configuration.md:105]: Just flagging the mismatch between the parameter's name and description. Could you confirm if this is intentional?

**Review by buraizu — APPROVED** [review]
Thanks, posted a question about the new parameter's description. Also, could you please confirm if this is ready to merge?

File: content/en/synthetics/private_locations/configuration.md (hunk 1 above)
Comment (HugoPct) [reply content/en/synthetics/private_locations/configuration.md:105]: message is indeed tests

Comment (HugoPct) [reply content/en/synthetics/private_locations/configuration.md:105]: good to go :)

**Comment by buraizu** [comment]
Checked in local build.

**Pull request merged**